src/app/applications/applications.controller.app.js

```diff
@@ -2,11 +2,12 @@
   'use strict';
 
   angular
-  .module('roboconf.applications')
+  .module('roboconf.applications', ['ngCropper'])
   .controller('SingleApplicationController', singleApplicationController);
 
-  singleApplicationController.$inject = ['rClient', '$scope', '$routeParams', '$window', 'rUtils'];
-  function singleApplicationController(rClient, $scope, $routeParams, $window, rUtils) {
+  singleApplicationController.$inject = ['rClient', '$scope', '$routeParams', '$window', '$timeout',
```

Comment:
Without $timeout(showCropper), the cropper is not displayed. It allows the loading of the Cropper.